tests/vars.sample.yaml

```diff
@@ -10,5 +10,8 @@ oc_header: |
 # External MariaDB IP for DB exports.
 external_mariadb_ip: CUSTOMIZE_THIS
 
+# External OVN DB IP for DB exports.
+external_ovndb_ip: CUSTOMIZE_THIS
```

Comment:
The PR looks good, we'll need to wire this into CI similarly as `external_mariadb_ip` to have a chance of passing CI.

https://review.rdoproject.org/cgit/rdo-jobs/tree/playbooks/data_plane_adoption/run_repo_tests.yaml#n12